Cargo.toml

```diff
@@ -49,9 +49,7 @@ lto = true
 
 [workspace.metadata.release]
 shared-version = true
-dev-version-ext = "beta.0"
 consolidate-commits = true
-consolidate-pushes = true
 tag-name = "v{{version}}"
```

Comment:
These got removed since they were for an older `cargo-release` version and are no longer supported